orgs-using-d.dd

```diff
@@ -159,6 +159,13 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING www.weka.io)
         )
     )
+    $(DORG GNEX, https://globalnet-ex.com, gnex.png,
+        Digital Marketing Solutions Company.,
+        $(LINK_ROW
+            $(FA_GITHUB gnexltd) $(FA_SEPARATOR)            
+            $(FA_HIRING https://globalnet-ex.com/employment/engineer)
```

Comment:
You need to remove `https://`, otherwise, the link doesn't work, because of how ddoc handles `:`.